### PR TITLE
Adding new attributes to our Pizza card

### PR DESCRIPTION
**Pull request by munzah-shah**
This PR does the following:

- [x] Updates the title of the page
- [x] Updates the attributes of Pizza

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
 
-    <title>Food Coupon page by Mahnoor | HTML and CSS</title>
+    <title>Food Coupon page | HTML and CSS</title>
```
Comment (munzah-shah) [inline index.html:9]: add "by Mahnoor" again

Comment (munzah-shah) [reply index.html:9]: no I don't want to

**Review by munzah-shah — COMMENTED** [review]
Sugggestion: revert back all changes.

**Review by mahnoorjanjua505 — CHANGES_REQUESTED** [review]
reverse the changes!

**Pull request merged**